src/main/sensors/rpm_filter.c

```diff
@@ -156,6 +156,7 @@ float rpmFilterDterm(int axis, float value)
 
 static float motorFrequency[MAX_SUPPORTED_MOTORS];
 
+
```

Comment:
Extra whitespace making this file added to the commits for no reason.

Reply:
This is fixed in the latest push